x-pack/plugins/apm/common/utils/as_mutable_array.ts

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export function asMutableArray<T extends Readonly<any>>(
```

Comment:
Can you add a comment to describe what this helper does?

Reply:
sure thing 👍